backend/src/main/java/com/example/cms/page/PageController.java

```diff
@@ -104,4 +104,10 @@ ResponseEntity<Void> modifyPageKeyWords(@PathVariable long id, @RequestBody Stri
         service.modifyKeyWordsField(id, keyWords);
         return ResponseEntity.noContent().build();
     }
+
+    @PutMapping("/{id}/handlers")
+    ResponseEntity<Void> addPageHandlers(@PathVariable long id, @RequestBody List<Long> pageHandlers) {
```

Comment:
Proszę o dodanie `List<Long> contactRequestHandlers` do `PageEditFormDto` i przypisywanie ich z metody do edycji `Page`, wtedy już z poziomu frontu też będzie działać bo na to jest przygotowany formularz